ruby/ql/lib/codeql/ruby/Regexp.qll

```diff
@@ -0,0 +1,143 @@
+/**
```

Comment:
Regarding the naming of this file: although the Ruby class is spelled `Regexp`, we've been using a capital E (`RegExp`) for the QL module/class names, for consistency between languages.

Reply:
Yes, I considered that, however, for JavaScript the file is named `javascript/ql/lib/semmle/javascript/Regexp.qll` too.